Ex_22/index.js

```diff
@@ -0,0 +1,13 @@
+"use striect"
+alert ('Hello, user!');
+let firstNum = Number(prompt ('Please, enter first number', 100));
+let secondNum = Number(prompt ('Please, enter second number', 100));
```

Comment:
Nice that you used function inside function. Number(prompt());
That's a very common practice in programming world across different programming languages!
We need to get used to this.
👍